src/drag-and-drop.js

```diff
@@ -0,0 +1,49 @@
+import ToDo from './todo.js';
+
+const toDo = new ToDo();
+let dragged;
+
+function allowDrop(ev) {
+  ev.preventDefault();
+}
+
+function drag(ev) {
+  dragged = this;
+  ev.dataTransfer.setData('text', this.outerHTML);
+}
+
+function sortItems() {
+  const sortedItems = document.getElementById('to-do-list').querySelectorAll('.task-item');
+  const itemsIndex = [];
+  sortedItems.forEach((item) => {
+    itemsIndex.push(item.children[1].id);
+  });
+
+  const tasks = toDo.localStorageTasks;
+  const temp = [];
+  for (let i = 0; i < tasks.length; i += 1) {
```

Comment:
- [ ] As requested by the previous reviewer, kindly implement the drag and drop functionality. It still doesn't work.